cmd/ebrelayer/main.go

```diff
@@ -206,8 +212,14 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 	zap.RedirectStdLog(sugaredLogger.Desugar())
 
 	// Initialize new Ethereum event listener
+	//validatorAddress, err := relayer.AddToKeyringWithMnemonic(cliContext.Keyring, validatorMoniker, mnemonic)
+	//if err != nil {
+	//	panic("could not add relayer to keyring")
+	//}
+	//
+	//valAddrAsFrom := sdk.ValAddress(validatorAddress.GetAddress())
 	ethSub, err := relayer.NewEthereumSub(cliContext, rpcURL, validatorMoniker, chainID, web3Provider,
-		contractAddress, privateKey, mnemonic, db, sugaredLogger)
+		contractAddress, privateKey, nil, db, sugaredLogger, "")
```

Comment:
If these parameters aren't needed, let's remove the params from `NewEthereumSub`.

Reply:
Added a comment; this needs to be addressed when we do replay